broker/message.pb.go

```diff
@@ -1,14 +1,15 @@
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
 // source: message.proto
 
-package mainflux
+package broker
```

Comment:
Please don't change this manually, but rather fix Makefile and use it to generate this code. I don't mind keeping `broker.go` and `messages.{proto, pb.go}` in the project root.